<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="hostnameLabel">
      <property name="text">
       <string notr="true">TextLabel</string>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>1</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="tStackedWidget" name="stackedWidget">
      <widget class="UserListPane" name="userList"/>
      <widget class="PasswordPane" name="passwordPage"/>
      <widget class="ReadyPane" name="readyPage"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserListPane</class>
   <extends>QWidget</extends>
   <header>userlistpane.h</header>
   <container>1</container>
   <slots>
    <signal>userSelected(QString,QString)</signal>
    <signal>reject()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">tstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PasswordPane</class>
   <extends>QWidget</extends>
   <header>passwordpane.h</header>
   <container>1</container>
   <slots>
    <signal>reject()</signal>
    <signal>accept(QString)</signal>
    <signal>sessionChanged(QString)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ReadyPane</class>
   <extends>QWidget</extends>
   <header>readypane.h</header>
   <container>1</container>
   <slots>
    <signal>reject()</signal>
    <signal>accept()</signal>
    <signal>sessionChanged(QString)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
